tasks/demo/admin/createPackTemplateAndDrop.sh

```diff
@@ -0,0 +1,140 @@
+sleep 0.5
+
+if [ -z $packTemplateDefaultValues ]; then
+    packTemplateName=$(generateString)
+    description=$(generateString)
+    maxSupply=$RANDOM
+    imageUrl="https://$(generateString).png"
+else
+    read -p "What is the pack template name? : " packTemplateName
+    read -p "What is the description? : " description
+    read -p "What is the max supply? : " maxSupply
+    read -p "what is the imageUrl? : " imageUrl
+fi
+
+echo "------------------- INFOS PACK TEMPLATE -------------------"
+echo "Pack template name: $packTemplateName"
+echo "Description: $description"
+echo "Max supply: $maxSupply"
+echo "Image url: $imageUrl"
+echo "-----------------------------------------------------------"
+
+# Get pack templates ids
+packTemplatesIDs=$(flow scripts execute scripts/mfl/packs/get_pack_template_ids.script.cdc | grep -o "[0-9]")
+currenPackTemplateID=$(echo "${packTemplatesIDs[*]}" | sort -nr | head -n1)
+[ -z $currenPackTemplateID ] && nextPackTemplateID=1 || nextPackTemplateID="$(($currenPackTemplateID+1))"
+echo "The pack template id will be : $nextPackTemplateID"
+
+# Drop creation
+read -p $'Do you want default values for the drop? (Press Enter for yes)\n' -n1 dropDefaultValues
+sleep 0.5
+
+if [ -z $dropDefaultValues ]; then
+    dropName=$(generateString)
+    price="$((1 + $RANDOM % 50)).00" # nbr between 1 and 50
+    packTemplateID=$nextPackTemplateID # link to the packTemplate above
+    maxTokensPerAddress=$((1 + $RANDOM % 20)) #nbr between 1 and 20
+else
+    read -p "What is drop name? : " dropName
+    read -p "What is the price of a pack? : " price
+    read -p "What is the packTemplate id? : " packTemplateID
+    read -p "what is the maxTokensPerAddress? : " maxTokensPerAddress
+fi
+
+read -p $'The drop is closed by default. Do you want to open it for whitelisted addresses (1) or to all (2) : \n' -n1 dropStatus
+echo ""
+if [ "$dropStatus" != 1 ] && [ "$dropStatus" != 2 ]; then
+    dropStatus=0
+fi
+
+echo "------------------- INFOS DROP -------------------"
+echo "Drop name: $dropName"
+echo "Price: $price"
+echo "Pack template id: $packTemplateID"
+echo "Max tokens per address: $maxTokensPerAddress"
+echo "Drop status: $dropStatus"
+echo "--------------------------------------------------"
+
+# Get drops ids
+dropIDs=$(flow scripts execute scripts/mfl/drops/get_ids.script.cdc | grep -o "[0-9]")
+currentDropID=$(echo "${dropIDs[*]}" | sort -nr | head -n1)
+[ -z $currentDropID ] && nextDropID=1 || nextDropID="$(($currentDropID+1))"
+echo "The drop id will be : $nextDropID"
+
+# Bob'account whitelisted or not
+read -p $'Do you want to whitelist bob\'s address? If so, what is the max number of packs he can buy? (Press Enter to pass)\n' bobNbrPacksWhitelist
+if [[ $bobNbrPacksWhitelist -gt $maxTokensPerAddress ]]; then
+    echo "Error - max number of packs ($bobNbrPacksWhitelist) must be smaller or equal to max tokens per address ($maxTokensPerAddress)"
+    exit 1
```

Comment:
instead of exiting here maybe you could loop until the answer is correct or the user press "q" to quit so you don't have to type everything again if you made a typo

Reply:
It could be the same for every steps, When you fill the pack details, ask if the info are correct or retype them and when you ask for the drops details, ask ask if the info are correct or retype them.